File: pinyougou_seckill_interface/src/main/java/com/pinyougou/seckill/service/SecKillService.java
package com.pinyougou.seckill.service;

import com.pinyougou.pojo.TbSeckillGoods;

import java.util.List;

/**
 * @author qinshiji
 * @data 2019/2/20 16:37
 */
public interface SecKillService {
    /**
     * 查询所有秒杀商品列表
     * @return
     */
    List<TbSeckillGoods> findList();

    /**
     * 查询秒杀商品
     * @param secKillGoodId
     * @return
     */
    TbSeckillGoods findOne(Long secKillGoodId);

    /**
     * 保存订单
     * @param userId
     * @param secKillGoodId
     */
    void saveOrder(String userId, Long secKillGoodId);
}

File: pinyougou_seckill_web/src/main/webapp/js/service/secKillService.js
//服务层
app.service('secKillService', function ($http) {
    //查询秒杀商品列表
    this.findList = function () {
        return $http.get("secKill/findList.do");
    }
    //查询秒杀商品详情
    this.findOne = function (secKillGoodId) {
        return $http.get("secKill/findOne.do?secKillGoodId="+secKillGoodId);
    }

    //对秒杀商品下单
    this.saveOrder = function (secKillGoodId) {
        return $http.get("secKill/saveOrder.do?secKillGoodId="+secKillGoodId)
    }


});
